adam/perception/_matcher.py

```diff
@@ -559,6 +559,26 @@ def subgraph_isomorphisms_iter(
         ):
             yield mapping
 
+    def subgraph_monomorphisms_iter(
+        self,
+        *,
+        collect_debug_statistics: bool = False,
+        debug_callback: Optional[Callable[[Any, Any], None]] = None,
+        initial_partial_match: Mapping[Any, Any] = immutabledict(),
+    ):
+        """Generator over monomorphisms from G2 to a subgraph of G1."""
+        # Declare that we are looking for graph-subgraph isomorphism.
+        self.test = "mono"
+        self.initialize(initial_partial_match=initial_partial_match)
```

Comment:
Can you refactor the shared code between this and `sub_graph_isomorphisms_iter` into a "private" method they both call?

Reply:
Actually, if `subgraph_isomorphisms_iter` is no longer used, let's just remove it altogether.

Reply:
@gabbard Done.